public/test/utils/regions/story.js

```diff
@@ -13,10 +16,28 @@ class Story {
         return group(1, this.dom, this);
     }
 
-    pacakgeSelector() {
-        return this.dom.querySelector('.front-selector');
+    getPackageInSelector(id) {
+        return $('.front-selector option[value=' + id + ']');
     }
-}
+
+    getPackageInSelectorText(id) {
+        return textInside('.front-selector option[value=\"' + id + '\"]');
+    }
+
+    selectedPackageName() {
+        return textInside('.front-selector option:selected');
+    }
+
+    manageLatestPackage(index) {
```

Comment:
I think this action is too complex, it selects a package from the list and clicks on action -> manage.

I'd split it, the test spec can call two methods doing a single action each